code/_onclick/click.dm

```diff
@@ -104,7 +104,7 @@
 		CtrlClickOn(A)
 		return 1
 
-	if(stat || paralysis || stunned || weakened)
+	if(stat || hasStatusEffect(src, SE_PARALYZED) || hasStatusEffect(src, SE_STUNNED) || hasStatusEffect(src, SE_WEAKENED))
```

Comment:
Knowing that something as lightweight as getter procs were causing issues in roach mob processing, I don't see this being a good idea. I wonder if it would be better to use bitflags to indicate whether a mob has a status or not.